test/test_ui.py

```diff
@@ -477,6 +475,85 @@ def test_pretend_move_album(self):
         self.assertIn(b'srcfile', self.i.path)
 
 
+class ExportTest(_common.TestCase):
+    def setUp(self):
+        super(ExportTest, self).setUp()
+
+        self.io.install()
+
+        self.libdir = os.path.join(self.temp_dir, b'testlibdir')
+        os.mkdir(self.libdir)
+
+        self.itempath = os.path.join(self.libdir, b'srcfile')
+        shutil.copy(os.path.join(_common.RSRC, b'full.mp3'), self.itempath)
+
+        # Add a file to the library but don't copy it in yet.
+        self.lib = library.Library(':memory:', self.libdir)
+        self.i = library.Item.from_path(self.itempath)
+        self.lib.add(self.i)
+        self.album = self.lib.add_album([self.i])
+
+        # Alternate destination directory.
+        self.otherdir = os.path.join(self.temp_dir, b'testotherdir')
+
+    def _move(self, query=(), dest=None, copy=False, album=False,
+              pretend=False, export=True):
+        commands.move_items(self.lib, dest, query, copy, album,
+                            pretend, export)
+
+    def test_move_item(self):
```

Comment:
`test_move_item` no longer really accurately describes what's going on here. Maybe you mean `test_export_item`? (And `move` -> `export` in other test names below?)